pymemcache/test/utils.py

```diff
@@ -43,6 +43,9 @@ def __init__(self,
         self.ignore_exc = ignore_exc
         self.encoding = encoding
 
+    def clear(self):
+        self._contents.clear()
```

Comment:
Let's add a docblock here because this isn't a standard memcache command.

Reply:
Sounds good. Let me know if I should add more for ex. (this method is used strictly for testing to increase convenience).